packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,248 @@
+# AutoComplete
+
+A SelectInput where you can type inside the input to filter through the items.
+
+<img width="215" alt="image" src="https://github.com/razorpay/blade/assets/30949385/63d95ab7-6231-4986-befb-1634269e7af3">
+
+## Design
+
+- [Figma - SelectInput & AutoComplete](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13590%3A171038&mode=dev)
+
+## Props
+
+> **Note**
+>
+> Below this section, I have added [examples of common AutoComplete usage](#usage). Check them out to understand usage of these props in details.
+
+| **Props**          | **Description**                                                                                                      | **Type**                                                        | **Default Value** |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------- | ----------------- |
+| label              | label of input                                                                                                       | string                                                          |                   |
+| accessibilityLabel | aria-label of input when label is not defined                                                                        | string                                                          |                   |
+| labelPosition      | Label Position like other inputs but with new option `inside-input`                                                  | 'top' \| 'left' \| 'inside-input'                               | 'top'             |
```

Comment:
can we mention here only in a note that this applies to select input as well and not other input elements or maybe a section explaining the rationale of this entire scenario and why only we want to support it on select input families and then link the section here?

Reply:
Yup I'll add a note